src/main/scala/wolfendale/nunjucks/globals/package.scala

```diff
@@ -24,7 +25,25 @@ package object globals {
     }
   }
 
+  val range: Value.Function = Value.Function{ parameters =>
+
+    State.pure {
+      if (parameters.parameters.forall(_.value.isInstanceOf[Number])) {
+        val (start, stop, step): (Value.Number, Value.Number, Value.Number) = parameters.parameters.length match {
+          case 1 => (Number(0), parameters.get(0).get.asInstanceOf[Number], Number(1))
+          case 2 => (parameters.get(0).get.asInstanceOf[Number], parameters.get(1).get.asInstanceOf[Number], Number(1))
+          case 3 => (parameters.get(0).get.asInstanceOf[Number], parameters.get(1).get.asInstanceOf[Number], parameters.get(2).get.asInstanceOf[Number])
+        }
```

Comment:
Would it be better to add something like `value` to `Numeric` to give the `Double` value of a `Numeric`? Then we don't need to do loads of type checking / casting. Might lead to odd cases though. That said, we do have double versions of the special cases of `Numeric` (`Double.NaN`, `Double.PositiveInfinity`, and `Double.NegativeInfinity`)